Assess metrics template (#352)
codatio/codat-docs: docs/assess/excel/enhanced-financials-report.md

---

> title: "Enhanced Financials Report"

Reviewer: ```suggestion
title: "Enhanced Financials Excel Report"
```

> description: "Download the enhanced financials report to Excel"

Reviewer: ```suggestion
description: "Download the Enhanced Financials report to Excel"
```

> sidebar_label: "Enhanced financials report"

Reviewer: ```suggestion
sidebar_label: "Enhanced Financials report"
```

createdAt: "2022-10-18T08:18:51.069Z"
updatedAt: "2023-05-05T12:00:00.000Z"
---

> Our enhanced financials Excel report is designed to give you the same data you would get via the API.  You can use this report to test some companies and familiarise yourself with the data before building to the equivalent API endpoints.

Reviewer: ```suggestion
Our Enhanced Financials Excel report gives you the same data you'd get from the [Enhanced Financials endpoints](/assess/enhanced-financials/overview), but in an Excel workbook.  You can use the report to test some companies and familiarise yourself with the data before building to the equivalent API endpoints.
```

> This report contains an output of the following feature:

Reviewer: ```suggestion
The report contains the output of the following features:
```

- [Enhanced profit and loss accounts](/assess/enhanced-financials/profit-and-loss-accounts)
- [Enhanced balance sheet accounts](/assess/enhanced-financials/balance-sheet-accounts)

## Prerequisites

> The report contains data based on the data types you have enabled. If a data type is missing, the data associated with it will be omitted.

Reviewer: ```suggestion
The Enhanced Financials Excel report contains data from the currently enabled data types. If a data type is not enabled, the data associated with it is omitted.
```

> This report requires an accounting integration with the following datatypes enabled:

Reviewer: ```suggestion
The report requires an accounting integration with the following data types enabled:
```

- Accounts
- Profit and Loss
- Balance Sheet

## Generating the report

> The Assess Report can be generated via API or through [Codat's Portal](/assess/portal/overview#reports).  The [Excel reports](/assess/excel/overview) page describes how to generate and download the report via API.

Reviewer: ```suggestion
You can generate the Enhanced Financials Excel report using the API or the Codat Portal (for details, see [Reports](/assess/portal/overview#reports)). The [Assess reports in Excel](/assess/excel/overview) article describes how to generate and download the report using the API.
```

## Using the report

> We recommend creating a pivot table and grouping by category Level 1-5 to replicate a financial statement view.  Users can also create formulas for financial ratios.

Reviewer: ```suggestion
To get the most out of the report, try creating a pivot table and grouping the data by category Level 1-5 to replicate a financial statement view. You can also create formulas to calculate financial ratios.
```

### Metrics template

> We have created a [Metrics Template](/documents/assess-metrics.xlsx) that allows users to paste data from the enhanced financials report to calculate key financial metrics and ratios.

Reviewer: ```suggestion
We've also created a [Metrics Template Excel file](/documents/assess-metrics.xlsx). You can paste data into this file from the Enhanced Financials report to calculate key financial metrics and ratios.
```
